### DISCUSSION TITLE
Offer vertical tab menu and make it scrollable

### DISCUSSION THREAD

**Discussion by PacoMPerez** [primefaces/community#1134] [PrimeNG]
Hi all, after succesfully implementing the tab menu `p-tabMenu`  I also add an option through `styleClass` where you can see the menu in vertical format but the `scrollable` function its not designed to work in vertical.

![Captura desde 2024-02-07 17-10-39](https://github.com/primefaces/community/assets/32292841/87f9edbd-faed-4e35-a34e-8e779467d121)

The conditions to show the buttons just consider scrollLeft and scrollWidth so for vertical version this will never works.

So please it would be greate to have native primeNg vertical tabmenu.

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for the feedback, it's a good enhancement request. Is it similar to megamenu/vertical https://primeng.org/megamenu#vertical ?

**Reply by PacoMPerez** [reply]
Hi, 

Thanks for the quick response, yes it would be great to have the orientation input on the tab menu component as well as making the scrollable function work for it.

**Reply by DeVanMipih** [reply]
Hello @cetincakiroglu ,

I would like know if this feature is planned and if this is the case, in which release ? 
Anyway, is there some link you can provide that allow me to follow the progress ?

Having a vertical tabmenu would be nice !

Best regards,
